core/SConscript.boardloader

```diff
@@ -124,6 +127,10 @@ if TREZOR_MODEL in ('T', 'R'):
     CPU_ASFLAGS = '-mthumb -mcpu=cortex-m4 -mfloat-abi=hard -mfpu=fpv4-sp-d16'
     CPU_CCFLAGS = '-mthumb -mcpu=cortex-m4 -mfloat-abi=hard -mfpu=fpv4-sp-d16 -mtune=cortex-m4 '
     CPU_MODEL   = 'STM32F427xx'
+elif TREZOR_MODEL in ('DISC1',):
+    CPU_ASFLAGS = '-mthumb -mcpu=cortex-m4 -mfloat-abi=hard -mfpu=fpv4-sp-d16'
+    CPU_CCFLAGS = '-mthumb -mcpu=cortex-m4 -mfloat-abi=hard -mfpu=fpv4-sp-d16 -mtune=cortex-m4 '
+    CPU_MODEL   = 'STM32F429xx'
```

Comment:
Would it still work if we merged the two branches, i.e. used STM32F427xx for disco1?

Reply:
The main thing here is that Disco uses LTDC that is not available on F427. STM HAL needs to be configured accordingly. It would most likely work if we set STM32F429xx for model T and R, but it doesn't feel right